packages/vue/src/components/molecules/SfModal/SfModal.vue

```diff
@@ -114,7 +114,7 @@ export default {
      */
     ariaLabelClose: {
```

Comment:
@martaradziszewska please remove these props and hardcore "Close Modal". In the future, we improve feature for translations static elements of components